src/main/java/by/techart/learn/LearnApplication.java

```diff
@@ -0,0 +1,13 @@
+package by.techart.learn;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class LearnApplication {
```

Comment:
What do you think about naming the application in a more concrete way? We can have different learning projects so it would be nice to have a more specific name to distinguish them. I can suggest using `UserApiApplication`

Reply:
Alright! Look's pretty fair.